apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestionTable.tsx

```diff
@@ -0,0 +1,781 @@
+  const handleStudentInfoCheckboxClick = () => {
+    const currentState = studentInfoCheckboxState;
+    const currentVisibility = table.getState().columnVisibility;
+
+    let newVisibility: VisibilityState;
+    if (currentState === 'checked') {
+      // Checked -> Unchecked (hide both)
+      newVisibility = {
+        ...currentVisibility,
+        user_or_group_name: false,
+        uid: false,
+      };
+    } else {
+      // Unchecked or Indeterminate -> Checked (show both)
+      newVisibility = {
+        ...currentVisibility,
+        user_or_group_name: true,
+        uid: true,
+      };
+    }
+
+    void setColumnVisibility(newVisibility);
+  };
+
+  const selectedRows = table.getSelectedRowModel().rows;
+  const selectedIds = selectedRows.map((row) => row.original.id);
+
+  // Set up handlers for edit points popovers
+  // FIXME: Clean up this useEffect once other pages handling point changes are also in React.
+  useEffect(() => {
+    /**
+     * Handle AJAX form submission for editing points
+     */
+    async function handlePointsFormSubmit(this: HTMLFormElement, event: Event) {
+      event.preventDefault();
+      const formData = new FormData(this);
+      // @ts-expect-error - It doesn't like converting FormData to an object
+      const postBody = new URLSearchParams(Object.fromEntries(formData.entries()));
+
+      try {
+        const response = await fetch(this.action || '', { method: 'POST', body: postBody });
+        if (response.status !== 200) {
+          console.error(response.status, response.statusText);
+          return;
+        }
+
+        const data = await response.json();
+
+        // Check for grading conflict
+        if (data?.conflict_grading_job_id) {
+          const modal = document.getElementById('grading-conflict-modal');
+          if (modal) {
+            const link = modal.querySelector<HTMLAnchorElement>('.conflict-details-link');
+            if (link && data.conflict_details_url) {
+              link.href = data.conflict_details_url;
+            }
+            // Show the modal using Bootstrap's Modal API
+            const bsModal = new window.bootstrap.Modal(modal);
+            bsModal.show();
+          }
+        }
+
+        // Invalidate and refetch the query to update the table
+        await queryClientInstance.invalidateQueries({
+          queryKey: ['instance-questions', urlPrefix, assessmentId, assessmentQuestion.id],
+        });
+      } catch (err) {
+        console.error('Error submitting form:', err);
+      }
+    }
+
+    /**
+     * Set up event listeners when popover is shown
+     */
+    function handlePopoverShown(this: Element) {
+      // Focus the first non-hidden input
+      const form = document.querySelector<HTMLFormElement>('form[name=edit-points-form]');
+      if (!form) {
+        return;
+      }
+      const input = form.querySelector<HTMLInputElement>('input:not([type="hidden"])');
+      if (input) {
+        input.focus();
+      }
```

Comment:
`assets/scripts/lib/popover.ts` has a `getPopoverContainerForTrigger` that could be called on `this` to simplify this and ensure that you're only looking in the relevant popover.